reason-to-ocaml.sh

```diff
@@ -14,12 +14,12 @@ fi
 
 fix_let_syntax() {
     extensions=("await" "some" "default" "ok" "assert")
-    for extension in ${extensions[@]}; do
+    for extension in "${extensions[@]}"; do
```

Comment:
Do we still have reasoan in the codebase or can we drop this/move this somewhere else?

Reply:
We have some old prs with reason. After the next push to clean up old pr's, let's remove this.